.gitignore

```diff
@@ -0,0 +1,41 @@
+# Gitignore c++ template from https://github.com/github/gitignore/blob/main/C%2B%2B.gitignore
+
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+*.smod
+
```

Comment:
we are not doing Fortran here, so don't worry about it :)
```suggestion
```

Reply:
sure :+1:  that was just copied from the template